### ISSUE TITLE
Hamburger icon is missing aria-label

### ISSUE THREAD

**Issue by EmmaDawsonDev** [EddieHubCommunity/EventCalendar#55]
The menubar uses a hamburger menu on mobile. However, it needs an aria-label so that it is accessible.
<img width="403" alt="Screenshot 2021-11-09 145020" src="https://user-images.githubusercontent.com/57045550/140936368-456ee88b-b340-4966-ac1f-e0990c1936b2.png">

**Comment by EmmaDawsonDev** [comment]
This is an issue with prime react components so I will close this issue.
